es-space-mgmt/main/default/classes/ReservationManager.cls-meta.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>49.0</apiVersion>
```

Comment:
You forgot to move that one to `50.0`.

Reply:
Ya, I had it fixed in the other PR! I also just fixed it :)